### ISSUE TITLE
Few questions and comments on the method and code

### ISSUE THREAD

**Issue by migmartinho** [clguo/SA-UNet#15]
Hi, I have a few questions and comments regarding the method and the code:

1. The URL for the DRIVE dataset seems to go to the STARE dataset.
2. How were the number of epochs, learning rate scheduling (lr=1e-3 for the first 100 epochs, then lr=1e-4 for the last 50 epochs), dropblock rate and batch size defined? Did you use some validation set, or were these values based on previous work? If so, could you point me to it?
3. The metrics reported in the paper were computed at the resized image level (592x592 px)?
4. Why was 592x592 px chosen as the input size to the SA U-Net model?
5. The size of the augmented STARE dataset is 260 (20 images, 4 base transformations per image, 3 random transformation per base transformation), not 256 as it is mentioned in the paper. The same goes for the DRIVE dataset.
6. Why did you decide to monitor accuracy for an imbalanced task?
7. Could you give an interpretation for the space attention module?
8. Did you perform transferability experiments between the two datasets to understand how robust is the model when binarizing vascular retinal images obtained in different settings than what it was trained on?

Thanks!
